File: _springboot0321/src/main/java/com/study/springboot/MyController.java
package com.study.springboot;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class MyController {
    @GetMapping("/vue")
    public String vue() {
        return "vue/index"; //호용님 저희 천천히... 즐겁게해요>< 아직 아무것도 안했답니다><
    }
}

File: kot0410/src/main/kotlin/com/study/springboot/controller/MyController.kt
package com.study.springboot.controller

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class MyController {
    @GetMapping("/")
    fun root(model: Model) {
        model.addAttribute("name", "테스트 이름입니다.")
    }
}